mage_ai/ai/llm_pipeline_wizard.py

```diff
@@ -491,7 +491,9 @@ async def async_generate_comment_for_block(self, block_content: str) -> str:
         chain = LLMChain(llm=self.llm, prompt=prompt_template)
         function_comments_json = await chain.arun(block_content=block_content)
         function_comments = json.loads(function_comments_json)
-        return self.__insert_comments_in_functions(block_content, function_comments)
+        return dict(
```

Comment:
you don't need to update it to return dict now.
The UI already integrates this API response format.